packages/chrome/src/styled/sheet/StyledSheetBody.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'chrome.sheet_body';
+
+export const StyledSheetBody = styled.div.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})`
+  flex: 1;
+  overflow-y: auto;
+  padding: 20px;
```

Comment:
Any particular reason not to use `theme` driven padding configuration? (e.g. [`StyledPanel.ts`](https://github.com/zendeskgarden/react-components/blob/main/packages/accordions/src/styled/accordion/StyledPanel.ts))

Reply:
Updated to utilize theme space base, from our discussion offline.